README.md

```diff
@@ -31,7 +31,30 @@ TODO: INSTALLATION AND SETUP
 
 ## Application Structure
 
-TODO: DOCUMENT APPLICATION STRUCTURE
+The application structure below is a starting point for the root project structure. This will be built upon as
+standards are implemented for deeper directories.
+
+*DEV: When proposing standards that affect the application structure, please update this tree.*
+
+
+```
+.
+├── bin                      # Build/Start scripts
+├── dist                     # The folder for the compiled output (ignored by git)
+├── docs                     # Documentation files for the project
```

Comment:
So all this works for me/consider this my approval since I preemptively closed my review.